api/kyverno/v1/image_verification_types.go

```diff
@@ -63,6 +63,10 @@ type ImageVerification struct {
 	// Repository is an optional alternate OCI repository to use for image signatures that match this rule.
 	// If specified Repository will override the default OCI image repository configured for the installation.
 	Repository string `json:"repository,omitempty" yaml:"repository,omitempty"`
+
+	// DigestMutate is an optional field which handles the tag-to-digest mutation for the provided image paths.
+	// Defaults to true.
```

Comment:
Please use kubebuilder flags for defaults and optional. See: https://book.kubebuilder.io/reference/markers/crd.html.

Once you have added those, make sure you generate the CRDs and also run `make codegen`.

Reply:
@Namanl2001 - please see prior use of flags in the CRD definition:

https://github.com/kyverno/kyverno/blob/a81a29ec4312ecb9a8ab830e29ce962bcfb80c59/api/kyverno/v1/image_verification_types.go#L19

The flags are used by the code generator for validation logic and defaults.

Reply:
so just adding `// +kubebuilder:validation:Optional` will work right @JimBugwadia?

Reply:
And default values, min, max, etc. where applicable. 

See available list at: https://book.kubebuilder.io/reference/markers/crd-validation.html.

Reply:
added 

```
// +kubebuilder:default:=true
// +kubebuilder:validation:Optional
```

Reply:
not sure why the tests are failing, error suggests running `make gen-helm` but there are no new changes on running it.